jest.config.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
   setupFilesAfterEnv: ["<rootDir>/test/setup.js"],
   snapshotSerializers: ["enzyme-to-json/serializer"],
   transform: {
-    "^.+\\.(jsx?|tsx?)$": "babel-jest",
-    "(?!^.+\\.(jsx?|tsx?)$)": "<rootDir>/test/transforms/file.js",
+    "^.+\\.(js|jsx|ts|tsx)$": "babel-jest",
+    "(?!^.+\\.(js|jsx|ts|tsx)$)": "<rootDir>/test/transforms/file.js",
```

Comment:
Apparently the patterns from before did not work properly. When I tried them with the SVG file transform, they did not properly match.

For reference, see:
https://github.com/facebook/create-react-app/blob/v4.0.3/packages/react-scripts/scripts/utils/createJestConfig.js#L43-L51